generators/kubernetes/templates/service.yml.ejs

```diff
@@ -35,4 +35,9 @@ spec:
   <%_ } _%>
   ports:
   - name: http
+  <%_ if (app.applicationType === 'microservice' || app.applicationType === 'uaa') { _%>
```

Comment:
Unless there is a really good reason for this, I'd prefer using the condition : `!app.serviceDiscoveryType || app.applicationType === 'uaa'` -> so it will be specific to UAA and will not impact the other option as the other options works well with Kubernetes

Reply:
@pascalgrimaud Thanks, I made your requested change.